sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusSessionReceiverOptions.cs

```diff
@@ -43,6 +43,11 @@ public int PrefetchCount
         /// <inheritdoc cref="ServiceBusReceiverOptions.Identifier"/>
         public string Identifier { get; set; }
 
+        /// <summary>
+        /// Number of messages to receive.
+        /// </summary>
+        public int BatchSize { get; set; } = 1;
```

Comment:
let's check that it's greater than 0 when set

Reply:
updated!